src/openrct2/ride/transport/Monorail.cpp

```diff
@@ -1019,8 +950,8 @@ static void paint_monorail_track_left_eighth_to_diag(
     if (index >= 0)
     {
         uint32_t imageId = ghost_train_track_pieces_left_eight_to_diag[direction][index] | session->TrackColours[SCHEME_TRACK];
-        const CoordsXY offset = ghost_train_track_pieces_left_eight_to_diag_offset[direction][index];
-        const CoordsXY bounds = ghost_train_track_pieces_left_eight_to_diag_bounds[direction][index];
+        const CoordsXY offset = GhostTrainTrackPiecesLeftEightToDiagBoxes[direction][index].offset;
```

Comment:
Dont use copies, use constant references, might as well declare a reference with the `BoundBoxXY` type and use it directly in `PaintAddImageAsParent`.

Reply:
That’s outside the scope of this PR, but might be a nice thing for a future refactor.